src/BloomExe/Edit/EditingView.cs

```diff
@@ -522,14 +520,9 @@ private void _browser1_OnBrowserClick(object sender, EventArgs e)
 					ge.Handled = false; //let gecko handle it
 					return;
 				}
-				else
-				{
-					ErrorReport.NotifyUserOfProblem("Bloom did not understand this link: " + anchor.Href);
-					ge.Handled = true;
-				}
-
+				ErrorReport.NotifyUserOfProblem("Bloom did not understand this link: " + anchor.Href);
```

Comment:
One or the other was redundant. It's shorter to remove the 'else', but perhaps more intent-revealing to remove the return.
One way I like to think of this sort of thing: an early return in a method typically means something a bit unexpected happened so that the method's usual task is not being completed. If/else is more appropriate when there are just two ways to accomplish a task, depending on something.
Another helpful question to ask yourself: if something extra needed to be added at the end of this method, does it feel more likely that it should be done in both cases or only if the condition is false?